Examples/BinaryBH/BinaryBHLevel.cpp

```diff
@@ -154,12 +156,15 @@ void BinaryBHLevel::specificPostTimeStep()
     // do puncture tracking on requested level
     if (m_p.track_punctures && m_level == m_p.puncture_tracking_level)
     {
-        CH_TIME("PunctureTracking");
-        // only do the write out for every coarsest level timestep
-        int coarsest_level = 0;
-        bool write_punctures = at_level_timestep_multiple(coarsest_level);
-        m_bh_amr.m_puncture_tracker.execute_tracking(m_time, m_restart_time,
-                                                     m_dt, write_punctures);
+        if (m_time != 0) // already done at t=0 in Main
```

Comment:
As discussed, if this guard can happen in the puncture tracker that would be nice, but fine if not.